### ISSUE TITLE
  All text in the resulting PNG is duplicated

### ISSUE THREAD

**Issue by HarryLonsdale** [gabelerner/canvg#344]
Hi. 
I have a similar problem:

http://stackoverflow.com/questions/28563073/when-using-canvg-to-convert-highchart-svg-into-png-all-text-appears-twice-how

**Comment by tomwayson** [comment]
I have the same problem. I'm using this example of how to convert c3 charts to canvas:

https://github.com/gonsakon/SVGtoPNG

Which inlines the c3 styles into the svg element before trying to use canvg to output to canvas. When I use the copy of the canvg files in that repo, it works fine, but when I try w/ the latest canvg files from this repo, I see duplicate text for every label on the chart (both x and y axis). 

I've included output svg (after styles have been inlined) below. If you run that in the [test page](http://gabelerner.github.io/canvg/examples/index.htm) you will see the issue.

There is [a solution](http://stackoverflow.com/a/29610373/656010) posted to the above question. Does it seem like that would work? Do you think that would break anything else? If not, would you like me to test and PR?

```xml
<svg width="500" height="320" style="overflow: hidden; font-style: normal; font-variant: normal; font-weight: normal; font-stretch: normal; font-size: 10px; line-height: normal; font-family: sans-serif;"><g transform="translate(50.5,4.5)"><text class="c3-text c3-empty" text-anchor="middle" dominant-baseline="middle" x="224" y="133" style="opacity: 0; display: none; -webkit-user-select: none; fill: rgb(128, 128, 128); font-size: 2em;"></text><rect class="c3-zoom-rect" width="448" height="266" style="opacity: 0; display: none;"></rect><g clip-path="url(http://localhost/code/git/SVGtoPNG/#c3-1430429564818-clip)" class="c3-regions" style="visibility: visible;"></g><g clip-path="url(http://localhost/code/git/SVGtoPNG/#c3-1430429564818-clip-grid)" class="c3-grid" style="visibility: visible;"><g class="c3-xgrid-focus" style="shape-rendering: crispEdges;"><line class="c3-xgrid-focus" x1="356" x2="356" y1="0" y2="266" style="visibility: hidden; display: none; fill: none; stroke: rgb(170, 170, 170); shape-rendering: crispEdges;"></line></g></g><g clip-path="url(http://localhost/code/git/SVGtoPNG/#c3-1430429564818-clip)" class="c3-chart"><g class="c3-event-rects c3-event-rects-single" style="fill-opacity: 0;"><rect class=" c3-event-rect c3-event-rect-0" x="-38.9" y="0" width="87.8" height="266" style="shape-rendering: crispEdges;"></rect><rect class=" c3-event-rect c3-event-rect-1" x="49.1" y="0" width="87.8" height="266" style="shape-rendering: crispEdges;"></rect><rect class=" c3-event-rect c3-event-rect-2" x="137.1" y="0" width="87.8" height="266" style="shape-rendering: crispEdges;"></rect><rect class=" c3-event-rect c3-event-rect-3" x="224.1" y="0" width="87.8" height="266" style="shape-rendering: crispEdges;"></rect><rect class=" c3-event-rect c3-event-rect-4" x="312.1" y="0" width="87.8" height="266" style="shape-rendering: crispEdges;"></rect><rect class=" c3-event-rect c3-event-rect-5" x="400.1" y="0" width="87.8" height="266" style="shape-rendering: crispEdges;"></rect></g><g class="c3-chart-bars"><g class="c3-chart-bar c3-target c3-target-data1" style="opacity: 1; pointer-events: none;"><g class=" c3-shapes c3-shapes-data1 c3-bars c3-bars-data1" style="cursor: pointer;"></g></g><g class="c3-chart-bar c3-target c3-target-data2" style="opacity: 1; pointer-events: none;"><g class=" c3-shapes c3-shapes-data2 c3-bars c3-bars-data2" style="cursor: pointer;"></g></g></g><g class="c3-chart-lines"><g class="c3-chart-line c3-target c3-target-data1" style="opacity: 1; pointer-events: none;"><g class=" c3-shapes c3-shapes-data1 c3-lines c3-lines-data1"><path class=" c3-shape c3-shape c3-line c3-line-data1" d="M5,232.59188034188034L93,136.33119658119656L181,192.9551282051282L268,23.083333333333314L356,164.64316239316238L444,108.01923076923075" fill="none" style="stroke: rgb(31, 119, 180); opacity: 1;"></path></g><g class=" c3-shapes c3-shapes-data1 c3-areas c3-areas-data1"><path class=" c3-shape c3-shape c3-area c3-area-data1" d="M 5 232.59188034188034" style="fill: rgb(31, 119, 180); opacity: 0.200000002980232;"></path></g><g class=" c3-selected-circles c3-selected-circles-data1"></g><g class=" c3-shapes c3-shapes-data1 c3-circles c3-circles-data1" style="cursor: pointer;"><circle class=" c3-shape c3-shape-0 c3-circle c3-circle-0" r="2.5" cy="232.59188034188034" cx="5" style="fill: rgb(31, 119, 180); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-1 c3-circle c3-circle-1" r="2.5" cy="136.33119658119656" cx="93" style="fill: rgb(31, 119, 180); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-2 c3-circle c3-circle-2" r="2.5" cy="192.9551282051282" cx="181" style="fill: rgb(31, 119, 180); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-3 c3-circle c3-circle-3" r="2.5" cy="23.083333333333314" cx="268" style="fill: rgb(31, 119, 180); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-4 c3-circle c3-circle-4" r="2.5" cy="164.64316239316238" cx="356" style="fill: rgb(31, 119, 180); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-5 c3-circle c3-circle-5" r="2.5" cy="108.01923076923075" cx="444" style="fill: rgb(31, 119, 180); opacity: 0; display: none;"></circle></g></g><g class="c3-chart-line c3-target c3-target-data2" style="opacity: 1; pointer-events: none;"><g class=" c3-shapes c3-shapes-data2 c3-lines c3-lines-data2"><path class=" c3-shape c3-shape c3-line c3-line-data2" d="M5,221.26709401709402L93,238.2542735042735L181,243.91666666666666L268,226.92948717948718L356,241.0854700854701L444,235.4230769230769" fill="none" style="stroke: rgb(255, 127, 14); opacity: 1;"></path></g><g class=" c3-shapes c3-shapes-data2 c3-areas c3-areas-data2"><path class=" c3-shape c3-shape c3-area c3-area-data2" d="M 5 221.26709401709402" style="fill: rgb(255, 127, 14); opacity: 0.200000002980232;"></path></g><g class=" c3-selected-circles c3-selected-circles-data2"></g><g class=" c3-shapes c3-shapes-data2 c3-circles c3-circles-data2" style="cursor: pointer;"><circle class=" c3-shape c3-shape-0 c3-circle c3-circle-0" r="2.5" cy="221.26709401709402" cx="5" style="fill: rgb(255, 127, 14); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-1 c3-circle c3-circle-1" r="2.5" cy="238.2542735042735" cx="93" style="fill: rgb(255, 127, 14); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-2 c3-circle c3-circle-2" r="2.5" cy="243.91666666666666" cx="181" style="fill: rgb(255, 127, 14); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-3 c3-circle c3-circle-3" r="2.5" cy="226.92948717948718" cx="268" style="fill: rgb(255, 127, 14); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-4 c3-circle c3-circle-4" r="2.5" cy="241.0854700854701" cx="356" style="fill: rgb(255, 127, 14); opacity: 0; display: none;"></circle><circle class=" c3-shape c3-shape-5 c3-circle c3-circle-5" r="2.5" cy="235.4230769230769" cx="444" style="fill: rgb(255, 127, 14); opacity: 0; display: none;"></circle></g></g></g><g class="c3-chart-arcs" transform="translate(224,128)"><text class="c3-chart-arcs-title" style="text-anchor: middle; opacity: 0; display: none; -webkit-user-select: none; dominant-baseline: middle; font-size: 1.3em;"></text><g class="c3-chart-arc c3-target c3-target-data1"><g class=" c3-shapes c3-shapes-data1 c3-arcs c3-arcs-data1"></g><text dy=".35em" class="" transform="translate(36.95367664538995,89.98791131250886)" style="opacity: 0; text-anchor: middle; pointer-events: none; display: none; -webkit-user-select: none; fill: rgb(255, 255, 255); font-size: 13px;">87.6%</text></g><g class="c3-chart-arc c3-target c3-target-data2"><g class=" c3-shapes c3-shapes-data2 c3-arcs c3-arcs-data2"></g><text dy=".35em" class="" transform="translate(-36.95367664538997,-89.98791131250886)" style="opacity: 0; text-anchor: middle; pointer-events: none; display: none; -webkit-user-select: none; fill: rgb(255, 255, 255); font-size: 13px;">12.4%</text></g></g><g class="c3-chart-texts"><g class="c3-chart-text c3-target c3-target-data1" style="opacity: 1; pointer-events: none;"><g class=" c3-texts c3-texts-data1"></g></g><g class="c3-chart-text c3-target c3-target-data2" style="opacity: 1; pointer-events: none;"><g class=" c3-texts c3-texts-data2"></g></g></g></g><g clip-path="url(http://localhost/code/git/SVGtoPNG/#c3-1430429564818-clip-grid)" class="c3-grid c3-grid-lines"><g class="c3-xgrid-lines"></g><g class="c3-ygrid-lines"></g></g><g class="c3-axis c3-axis-x" clip-path="url(http://localhost/code/git/SVGtoPNG/#c3-1430429564818-clip-xaxis)" transform="translate(0,266)" style="visibility: visible; opacity: 1;"><text class="c3-axis-x-label" transform="" dy="-0.5em" dx="-0.5em" x="448" style="text-anchor: end; -webkit-user-select: none;"></text><g class="tick" transform="translate(5, 0)" style="opacity: 1;"><line y2="6" x1="0" x2="0" style="fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: block; -webkit-user-select: none;"><tspan x="0" dy=".71em">0</tspan></text></g><g class="tick" transform="translate(93, 0)" style="opacity: 1;"><line y2="6" x1="0" x2="0" style="fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: block; -webkit-user-select: none;"><tspan x="0" dy=".71em">1</tspan></text></g><g class="tick" transform="translate(181, 0)" style="opacity: 1;"><line y2="6" x1="0" x2="0" style="fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: block; -webkit-user-select: none;"><tspan x="0" dy=".71em">2</tspan></text></g><g class="tick" transform="translate(268, 0)" style="opacity: 1;"><line y2="6" x1="0" x2="0" style="fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: block; -webkit-user-select: none;"><tspan x="0" dy=".71em">3</tspan></text></g><g class="tick" transform="translate(356, 0)" style="opacity: 1;"><line y2="6" x1="0" x2="0" style="fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: block; -webkit-user-select: none;"><tspan x="0" dy=".71em">4</tspan></text></g><g class="tick" transform="translate(444, 0)" style="opacity: 1;"><line y2="6" x1="0" x2="0" style="fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: block; -webkit-user-select: none;"><tspan x="0" dy=".71em">5</tspan></text></g><path class="domain" d="M0,6V0H448V6" style="fill: none; stroke: rgb(0, 0, 0);"></path></g><g class="c3-axis c3-axis-y" clip-path="url(http://localhost/code/git/SVGtoPNG/#c3-1430429564818-clip-yaxis)" transform="translate(0,0)" style="visibility: visible; opacity: 1;"><text class="c3-axis-y-label" transform="rotate(-90)" dy="1.2em" dx="-0.5em" x="0" style="text-anchor: end; -webkit-user-select: none;"></text><g class="tick" transform="translate(0,250)" style="opacity: 1;"><line x2="-6" style="fill: none; stroke: rgb(0, 0, 0);"></line><text x="-9" y="0" style="text-anchor: end; -webkit-user-select: none;"><tspan x="-9" dy="3">0</tspan></text></g><g class="tick" transform="translate(0,222)" style="opacity: 1;"><line x2="-6" style="fill: none; stroke: rgb(0, 0, 0);"></line><text x="-9" y="0" style="text-anchor: end; -webkit-user-select: none;"><tspan x="-9" dy="3">50</tspan></text></g><g class="tick" transform="translate(0,193)" style="opacity: 1;"><line x2="-6" style="fill: none; stroke: rgb(0, 0, 0);"></line><text x="-9" y="0" style="text-anchor: end; -webkit-user-select: none;"><tspan x="-9" dy="3">100</tspan></text></g><g class="tick" transform="translate(0,165)" style="opacity: 1;"><line x2="-6" style="fill: none; stroke: rgb(0, 0, 0);"></line><text x="-9" y="0" style="text-anchor: end; -webkit-user-select: none;"><tspan x="-9" dy="3">150</tspan></text></g><g class="tick" transform="translate(0,137)" style="opacity: 1;"><line x2="-6" style="fill: none; stroke: rgb(0, 0, 0);"></line><text x="-9" y="0" style="text-anchor: end; -webkit-user-select: none;"><tspan x="-9" dy="3">200</tspan></text></g><g class="tick" transform="translate(0,109)" style="opacity: 1;"><line x2="-6" style="fill: none; stroke: rgb(0, 0, 0);"></line><text x="-9" y="0" style="text-anchor: end; -webkit-user-select: none;"><tspan x="-9" dy="3">250</tspan></text></g><g class="tick" transform="translate(0,80)" style="opacity: 1;"><line x2="-6" style="fill: none; stroke: rgb(0, 0, 0);"></line><text x="-9" y="0" style="text-anchor: end; -webkit-user-select: none;"><tspan x="-9" dy="3">300</tspan></text></g><g class="tick" transform="translate(0,52)" style="opacity: 1;"><line x2="-6" style="fill: none; stroke: rgb(0, 0, 0);"></line><text x="-9" y="0" style="text-anchor: end; -webkit-user-select: none;"><tspan x="-9" dy="3">350</tspan></text></g><g class="tick" transform="translate(0,24)" style="opacity: 1;"><line x2="-6" style="fill: none; stroke: rgb(0, 0, 0);"></line><text x="-9" y="0" style="text-anchor: end; -webkit-user-select: none;"><tspan x="-9" dy="0">400</tspan></text></g><path class="domain" d="M-6,1H0V266H-6" style="fill: none; stroke: rgb(0, 0, 0);"></path></g><g class="c3-axis c3-axis-y2" transform="translate(448,0)" style="visibility: hidden; opacity: 1; display: none;"><text class="c3-axis-y2-label" transform="rotate(-90)" dy="-0.5em" dx="-0.5em" x="0" style="text-anchor: end; display: none; -webkit-user-select: none;"></text><g class="tick" transform="translate(0,266)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0</tspan></text></g><g class="tick" transform="translate(0,240)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0.1</tspan></text></g><g class="tick" transform="translate(0,213)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0.2</tspan></text></g><g class="tick" transform="translate(0,187)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0.3</tspan></text></g><g class="tick" transform="translate(0,160)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0.4</tspan></text></g><g class="tick" transform="translate(0,134)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0.5</tspan></text></g><g class="tick" transform="translate(0,107)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0.6</tspan></text></g><g class="tick" transform="translate(0,81)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0.7</tspan></text></g><g class="tick" transform="translate(0,54)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0.8</tspan></text></g><g class="tick" transform="translate(0,28)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">0.9</tspan></text></g><g class="tick" transform="translate(0,1)" style="opacity: 1; display: none;"><line x2="6" y2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text x="9" y="0" style="text-anchor: start; display: none; -webkit-user-select: none;"><tspan x="9" dy="3" style="display: none;">1</tspan></text></g><path class="domain" d="M6,1H0V266H6" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></path></g></g><g transform="translate(20.5,320.5)" style="visibility: hidden; display: none;"><g clip-path="url(http://localhost/code/git/SVGtoPNG/#c3-1430429564818-clip-subchart)" class="c3-chart" style="display: none;"><g class="c3-chart-bars" style="display: none;"></g><g class="c3-chart-lines" style="display: none;"></g></g><g clip-path="url(http://localhost/code/git/SVGtoPNG/#c3-1430429564818-clip)" class="c3-brush" style="pointer-events: all; -webkit-tap-highlight-color: rgba(0, 0, 0, 0); display: none;"><rect class="background" x="0" width="478" height="0" style="visibility: hidden; cursor: crosshair; display: none;"></rect><rect class="extent" x="0" width="0" height="0" style="cursor: move; display: none; fill-opacity: 0.1;"></rect><g class="resize e" transform="translate(0,0)" style="cursor: ew-resize; display: none;"><rect x="-3" width="6" height="0" style="visibility: hidden; display: none;"></rect></g><g class="resize w" transform="translate(0,0)" style="cursor: ew-resize; display: none;"><rect x="-3" width="6" height="0" style="visibility: hidden; display: none;"></rect></g></g><g class="c3-axis-x" transform="translate(0,0)" clip-path="url(http://localhost/code/git/SVGtoPNG/#c3-1430429564818-clip-xaxis)" style="opacity: 1; display: none;"><g class="tick" transform="translate(5, 0)" style="opacity: 1; display: none;"><line y2="6" x1="0" x2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: none; -webkit-user-select: none;"><tspan x="0" dy=".71em" style="display: none;">0</tspan></text></g><g class="tick" transform="translate(93, 0)" style="opacity: 1; display: none;"><line y2="6" x1="0" x2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: none; -webkit-user-select: none;"><tspan x="0" dy=".71em" style="display: none;">1</tspan></text></g><g class="tick" transform="translate(181, 0)" style="opacity: 1; display: none;"><line y2="6" x1="0" x2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: none; -webkit-user-select: none;"><tspan x="0" dy=".71em" style="display: none;">2</tspan></text></g><g class="tick" transform="translate(268, 0)" style="opacity: 1; display: none;"><line y2="6" x1="0" x2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: none; -webkit-user-select: none;"><tspan x="0" dy=".71em" style="display: none;">3</tspan></text></g><g class="tick" transform="translate(356, 0)" style="opacity: 1; display: none;"><line y2="6" x1="0" x2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: none; -webkit-user-select: none;"><tspan x="0" dy=".71em" style="display: none;">4</tspan></text></g><g class="tick" transform="translate(444, 0)" style="opacity: 1; display: none;"><line y2="6" x1="0" x2="0" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></line><text y="9" x="0" style="text-anchor: middle; display: none; -webkit-user-select: none;"><tspan x="0" dy=".71em" style="display: none;">5</tspan></text></g><path class="domain" d="M0,6V0H448V6" style="display: none; fill: none; stroke: rgb(0, 0, 0);"></path></g></g><g transform="translate(0,300)"><g class=" c3-legend-item c3-legend-item-data1" style="visibility: visible; cursor: pointer; font-size: 12px; opacity: 1;"><text x="213" y="9" style="pointer-events: none; -webkit-user-select: none;">data1</text><rect class="c3-legend-item-event" x="199" y="-5" width="56" height="18" style="fill-opacity: 0;"></rect><rect class="c3-legend-item-tile" x="199" y="0" width="10" height="10" style="pointer-events: none; fill: rgb(31, 119, 180); shape-rendering: crispEdges;"></rect></g><g class=" c3-legend-item c3-legend-item-data2" style="visibility: visible; cursor: pointer; font-size: 12px;"><text x="269" y="9" style="pointer-events: none; -webkit-user-select: none;">data2</text><rect class="c3-legend-item-event" x="255" y="-5" width="46" height="18" style="fill-opacity: 0;"></rect><rect class="c3-legend-item-tile" x="255" y="0" width="10" height="10" style="pointer-events: none; fill: rgb(255, 127, 14); shape-rendering: crispEdges;"></rect></g></g></svg>
```

**Comment by tomwayson** [comment]
Actually, I was **not** using the latest files from this repo when I saw that problem - I was using the ones hosted in the `gh-pages` branch as per the instructions on the README:

```html
<script type="text/javascript" src="http://gabelerner.github.io/canvg/rgbcolor.js"></script> 
<script type="text/javascript" src="http://gabelerner.github.io/canvg/StackBlur.js"></script>
<script type="text/javascript" src="http://gabelerner.github.io/canvg/canvg.js"></script> 
```

However, after downloading the master branch of this repo and using the files locally, I did not see the problem.

I now see that this issue was fixed in d7f438726f020b6ed32be5350d54a29b1c67d3f6 which was pushed *after* the most recent commit to gh-pages.

@gabelerner it appears that gh-pages is out of date and needs to be updated.
